diff --git a/config.def.h b/config.def.h
index defa426..c74228a 100644
--- a/config.def.h
+++ b/config.def.h
@@ -2,12 +2,12 @@
 
 /* appearance */
 static const char font[]        = "monospace:size=9";
-static const char* normbgcolor  = "#222222";
-static const char* normfgcolor  = "#cccccc";
-static const char* selbgcolor   = "#555555";
-static const char* selfgcolor   = "#ffffff";
-static const char* urgbgcolor   = "#111111";
-static const char* urgfgcolor   = "#cc0000";
+static const char* normbgcolor  = "#11151c";
+static const char* normfgcolor  = "#599cab";
+static const char* selbgcolor   = "#245361";
+static const char* selfgcolor   = "#d3ebe9";
+static const char* urgbgcolor   = "#11151c";
+static const char* urgfgcolor   = "#c23127";
 static const char before[]      = "<";
 static const char after[]       = ">";
 static const char titletrim[]   = "...";
